src/main/java/carpet/mixins/CoralFanBlock_renewableCoralMixin.java

```diff
@@ -0,0 +1,82 @@
+package carpet.mixins;
+
+import carpet.CarpetSettings;
+import carpet.fakes.CoralFeatureInterface;
+import net.minecraft.core.BlockPos;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.tags.FluidTags;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.BaseCoralPlantTypeBlock;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.BonemealableBlock;
+import net.minecraft.world.level.block.CoralFanBlock;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.levelgen.feature.CoralClawFeature;
+import net.minecraft.world.level.levelgen.feature.CoralFeature;
+import net.minecraft.world.level.levelgen.feature.CoralMushroomFeature;
+import net.minecraft.world.level.levelgen.feature.CoralTreeFeature;
+import net.minecraft.world.level.levelgen.feature.configurations.NoneFeatureConfiguration;
+import net.minecraft.world.level.material.MaterialColor;
+import org.spongepowered.asm.mixin.Mixin;
+
+import java.util.Random;
+
+@Mixin(CoralFanBlock.class)
+public abstract class CoralFanBlock_renewableCoralMixin implements BonemealableBlock
```

Comment:
Looks like this class was completely recreated...
Line endings, I'd guess? Or something
(https://github.com/altrisi/fabric-carpet/compare/mojmap...fxmorin:mojmap-class-names#diff-280cecdaf4cb09cc4e0e4ecc0114774572ab5662ec3868ff725b716cf401fbf7)